src/routes/_header.svelte

```diff
@@ -16,15 +16,23 @@
 <script lang="ts">
   import { namespace } from '$lib/stores/namespace';
   import DataConvertorStatus from '$lib/components/data-convertor-status.svelte';
+  import NavigationLink from './_navigation-link.svelte';
   export let user: { name?: string; email?: string; picture?: string } = {};
 </script>
 
 <div
   id="header"
-  class="h-16 w-full px-6 flex flex-row items-center justify-between border-b-2"
+  class="h-16 w-full px-6 flex flex-row items-center justify-between bg-gray-900 border-b-2"
```

Comment:
Looking at the mock up—your spacing is off. You probably want to use a [gap](https://tailwindcss.com/docs/gap) in the flexbox.